src/sdk/pynni/nni/nas/pytorch/fixed.py

```diff
@@ -33,6 +34,32 @@ def __init__(self, model, fixed_arc, strict=True):
             raise RuntimeError("Unexpected keys found in fixed architecture: {}.".format(fixed_arc_keys - mutable_keys))
         if mutable_keys - fixed_arc_keys:
             raise RuntimeError("Missing keys in fixed architecture: {}.".format(mutable_keys - fixed_arc_keys))
+        self._fixed_arc = self._convert_human_readable_architecture(self._fixed_arc)
+
+    def _convert_human_readable_architecture(self, human_arc):
+        result_arc = {k: to_list(v) for k, v in human_arc.items()}  # there could be tensors, numpy arrays, etc.
+        # First, convert non-list to list.
+        result_arc = {k: v if isinstance(v, list) else [v] for k, v in result_arc.items()}
+        # Second, infer which ones are multi-hot arrays and which ones are in human-readable format.
+        # This is non-trivial, since if an array in [0, 1], we cannot know for sure it means [false, true] or [true, true].
+        # Here, we assume an multihot array has to be a boolean array or a float array and matches the length.
```

Comment:
in what case the multihot array is float?

Reply:
so the exported final arch could be human readable or multihot?

Reply:
Yes. We can at least be backward-compatible on that.
Multi-hot array is float if it's a weighted-sum of candidates. By current design, we allow final architectures to be a mixture of conv3x3 and con5x5, for example.